app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+# rubocop:disable Style/ClassAndModuleChildren
+class Api::V1::UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
```

Comment:
where do you use it?

Reply:
Used when tested, not used at now, delete?